OracleSiebel/OEL/build/multi-stage/cgw/dockerfile

```diff
@@ -0,0 +1,141 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: Dec, 2019
+# Authors: andreas.thieltges@oracle.com and duncan.ford@oracle.com
+# Description: Build Siebel SAI container
+#
+# Notes: This build is execute from the build-all-multistage script
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+# Globals - these are overridden by the calling script
+
+ARG VERSION=19.03
+ARG BUILD_DATE=2019-04-30
+
+FROM  oraclelinux:latest as temp
```

Comment:
Please switch to `oraclelinux:7-slim` or at least `oraclelinux:7` as `oraclelinux:latest` will switch to 8 fairly soon.

Reply:
Done